### DISCUSSION TITLE
how to test type-graphql

### DISCUSSION THREAD

**Discussion by tanay-pingalkar** [MichalLytek/type-graphql#855] [Q&A]
i  am using type-graphql, is there anything in type-graphql like this `import { makeExecutableSchema } from "graphql-tools";`
or how can i use make ExecutableSchema in type-graphql or type-graphql's buildSchema will works fine.

**Comment by MichalLytek** [comment]
> type-graphql's buildSchema will works fine

Have you tried that before asking this question? 😅

**Reply by tanay-pingalkar** [reply]
> > type-graphql's buildSchema will works fine
> 
> Have you tried that before asking this question?

yes i have tried but it was not working, but now it is working fine, dont know why. I made a seperate util to build schema and use it in main file and test file. And it works.
